### ISSUE TITLE
Firefox support

### ISSUE THREAD

**Issue by 2br-2b** [2br-2b/datafaux#2]

**Comment by xR2** [comment]
If you check the .crx file of the old version (noiszy) on the a [compatibility site](https://www.extensiontest.com/) it says fully compatible.

So the fork should be too. Unfortunately I am not that _code-savy_.

**Comment by 2br-2b** [comment]
Yes - I know how to do it, I just need to take the time to actually *do* it.

**Comment by 2br-2b** [comment]
I probably should have made that more clear, but I'll try to get this on the AMO soon

**Comment by nigozeroichi** [comment]
Hi, guys, I'm not have a great JS background, so I want to now if it's possible to use datafaux in Firefox and, if so, how? Thank you very much.
